x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/dataframe/DataFrameAnalyticsConfigTests.java

```diff
@@ -468,24 +451,6 @@ public void testToXContent_GivenAnalysisWithRandomizeSeedAndVersionIsCurrent() t
         }
     }
 
-    public void testToXContent_GivenAnalysisWithRandomizeSeedAndVersionIsBeforeItWasIntroduced() throws IOException {
```

Comment:
This needs to stay.  The 7.5 here is embedded in the config document, not the node version.  Unless we do something else 8.x _will_ still have to cope with configs created in 7.5 and before.